Update User guide. (#198)
eu-cdse/documentation: Applications/Browser.md

Here you can create a custom *R/G/B* composite or _Index_ (band ratio, normalized difference index) by dragging and dropping the Sentinel-2 bands into the appropriate circles or use the _Custom script_ functionality to insert a piece of JavaScript code.

#### Changing Configurations

> You can also change the configuration to create your own layers and visualise. A configuration instance acts as a separate WMS/WMTS/WFS/WCS service and each can be configured to provide a certain set of layers with different settings. It is therefore possible to create multiple configuration instances each with a different set of layers fulfilling various needs. Configuration instances may contain an arbitrary number of layers, which are configurable using the settings defined above, such as MAXCC, TIME, the max area limitation, etc. Visualization using a predefined data product or a custom script is configured for each layer. The configuration instance itself also has some global settings for default values on all layers, like image quality. 

Reviewer: ```suggestion
You can change the configuration to create your own layers and visualizations. A configuration instance acts as a separate WMS/WMTS/WFS/WCS service and each can be configured to provide a certain set of layers with different settings. It is therefore possible to create multiple configuration instances each with a different set of layers fulfilling various needs. Configuration instances may contain an arbitrary number of layers, which are configurable using the settings defined above, such as cloud coverage, time range, etc. Visualization using a predefined visualization option or a custom script is configured for each layer. The configuration instance itself also has some global settings for default values on all layers, like image quality. 
```

> the max area limitation,

How is this possible?

> You can create your own confuguration by accessing your Dashboard from the dropdown Menu next to the Username. In the panel on the left, you will find "Configuration Utility" where you can create, modify and delete your instances. Once you have created a configuration, you can open it in the Copernicus Browser. Alternatively, you can change the configurations directly on the browser by selecting your configuration from the dropdown menu as shown in the figure.

Reviewer: ```suggestion
You can create your own configuration by accessing the Dashboard from the dropdown menu under your username or directly [here](https://shapps.dataspace.copernicus.eu/dashboard/). In the panel on the left, you will find "Configuration Utility" where you can create, modify and delete your instances. Once you have created a configuration, you can open it in the Copernicus Browser. Alternatively, you can change the configurations directly in the Copernicus Browser by selecting your configuration from the dropdown menu as shown in the figure.
```

#### Changing Configurations

> You can change the configuration to create your own layers and visualizations. A configuration instance acts as a separate WMS/WMTS/WFS/WCS service and each can be configured to provide a certain set of layers with different settings. It is therefore possible to create multiple configuration instances each with a different set of layers fulfilling various needs. Configuration instances may contain an arbitrary number of layers, which are configurable using the settings defined above, such as cloud coverage, time range, etc. Visualization using a predefined visualization option or a custom script is configured for each layer. The configuration instance itself also has some global settings for default values on all layers, like image quality. 

Reviewer: ```suggestion
You can change the configuration to create your own layers and visualizations. A configuration instance acts as a separate WMS/WMTS/WFS/WCS service and each can be configured to provide a certain set of layers with different settings. It is therefore possible to create multiple configuration instances, each providing a different set of layers for different needs. The configuration instances can contain any number of layers that can be configured with the settings defined above, e.g., cloud coverage, time range, etc. Each visualizations is based on a predefined visualization option or a custom script. The configuration instance itself also has some global settings for default values on all layers, suach as image quality. 
```

> You can create your own configuration by accessing the Dashboard from the dropdown menu under your username or directly [here](https://shapps.dataspace.copernicus.eu/dashboard/). In the panel on the left, you will find "Configuration Utility" where you can create, modify and delete your instances. Once you have created a configuration, you can open it in the Copernicus Browser. Alternatively, you can change the configurations directly in the Copernicus Browser by selecting your configuration from the dropdown menu as shown in the figure.

Reviewer: ```suggestion
You can create your own configuration by accessing the Dashboard via the drop-down menu under your username or directly [here](https://shapps.dataspace.copernicus.eu/dashboard/). In the panel on the left you will find "Configuration Utility" where you can create, modify and delete your instances. Once you have created a configuration, you can open it in the Copernicus Browser. Alternatively, you can change the configurations directly in the Copernicus Browser by selecting your configuration from the drop-down menu as shown in the figure.
```

<img src="_images/Filters.png" width="450">

Fig 12: Data filters and parameters

#### Visualize the search result

Once you have found a product, you can visualize the results in two ways: either by directly selecting the viszualize button ( ![visualize](_images/icons/VisualizeButton.png) ) in the sidebar or by selecting the visualize button in the results panel on the map. You can open the results panel by clicking on one of the displayed tile footprints on the map.

![Search tab](_images/VisualizeTile.png)

> Fig 13: Product metadata and visualize button

Reviewer: Can you please change this to <img src...> and then add a space between the image and the figure? The Figure title is currently not displayed nicely.

<img src="_images/ProductDownload.png" width="450">

Fig 14: Product download (in progress) with Download product and cancel button highlighted

#### Download single files

A "product" refers to a directory containing a collection of information such as metadata, product information, the satellite image and quality data, auxiliary data and more. If downloading all of this data is not of your interest, you can opt to download single files within this directory but clicking on the single file download icon (![SFD](_images/icons/SingleFileDownload.png)). Depending on the product you are trying to download, you will get a list of files within the directory and the  ![download](_images/icons/DownloadIcon.png) icon next to them to download each file individually.

### Add Product to Workspace

To avoid downloading all the products, you can add the product to your workspace and give as input to your workflow on the Copernicus Dataspace Ecosystem directly. The main objective of the workspace is to:

- Further process single products (with various options to select the algorithm/processing required for your application).

> 

Reviewer: please remove the empty line here

- Keep track of the processed products (status, finished/queued products)

> To do this, click on the Add to Workspace icon (![workspace](_images/icons/AddToWorkspace.png)) appearing in your search result.  The workspace can be accessed bu clicking on the dropdown menu next to your user name and clicking on Workspace. 

Reviewer: ```suggestion
To do this, click on the Add to Workspace icon (![workspace](_images/icons/AddToWorkspace.png)) appearing in your search result.  The workspace can be accessed by clicking on the dropdown menu under your username or directly [here](https://dataspace.copernicus.eu/workspace/).
```

> <img src="_images/User_DropDown.png" width="450">

Reviewer: ```suggestion
<img src="_images/User_DropDown.png" width="300">
```

Use the ![location](_images/icons/Location.png) icon to mark a location and re-centre to the **Point of Interest(POI)**

> 

Reviewer: Please remove the empty line here and link the next sentence to the one before (no line break). If you click on the file preview here you are able to see that it might get otherwise rendered quite ugly.

![image](https://github.com/eu-cdse/documentation/assets/60705209/79f817c7-67ff-4d30-9800-dfc9f9e61a94)

> Once you have selected the Area of interest, depending on the type of data you are looking at, you can view the spectral signature of the region with **Spectral Explorer**(![spectral](_images/icons/Spectral_Explorer.png)) or, in case of indices, look at the change in value over time with **Statistical Info** (![timeseries](_images/icons/HistIcon.png)).

Reviewer: ```suggestion
Once you have selected the AOI, depending on the type of data you are looking at, you can view the spectral signature of the region using the **Spectral Explorer**(![spectral](_images/icons/Spectral_Explorer.png)) or, in case of indices, look at the change in value over time using the **Statistical Info** (![timeseries](_images/icons/HistIcon.png)) feature.
```

![Timelapse](_images/Timelapse.png)
Fig 16: Browser timelapse tool with settings highlighted

### Statistical Ananlysis

> There are quite a lot of statistical analyses that can be done in the Browser itself. Depending on the type of data you are looking at, you can look at the distribution of the pixel values, view the spectral signature of the region using the **Spectral Explorer**(![spectral](_images/icons/Spectral_Explorer.png)) or, in case of indices, look at the change in value over time using the **Statistical Info** (![timeseries](_images/icons/HistIcon.png)) feature. 

Reviewer: ```suggestion
There are quite a lot of statistical analyses that can be done in the Browser itself. Depending on the type of data you are looking at, you can look at the distribution of the pixel values, view the spectral signature of the region using the **Spectral Explorer**(![spectral](_images/icons/Spectral_Explorer.png)) or, in case of indices, look at its change over time using the **Statistical Info** (![timeseries](_images/icons/HistIcon.png)) feature. 
```

2. Click on the **Spectral Explorer** icon (![spectral](_images/icons/Spectral_Explorer.png)) within the AOI tool.

<img src="_images/Browser_Spectral_Explorer.png" width="750">

Fig. 18: Screenshot of steps to follow to see an example of spectral signature of open water.

By doing this, you can see a graph pop up with the spectral signature (associated with specific chemical composition) averaged over all the pixels within this box. This helps to compare the spectral signature of the scene (in light green) with other known signatures.

<img src="_images/Spectral_Explorer.png" width="450">

Fig. 19: Example of spectral signatures plotted in comparison to other known signatures labelled in the bottom of the graph.

#### Time Series

With the **Statistical Info** (![timeseries](_images/icons/HistIcon.png)) feature within the AOI tool, you can see how the value of an index has changed in time. To see this, it is necessary to choose a visualisation which give an index as output, (e.g., NDVI) or any single band. You can follow these steps to see an example Time Series of the NDVI of a single agricultural field. 
 

>  1. Select the Normalized Difference Vegetation Index (NDVI) layer to visualise the [scene](https://sentinelshare.page.link/vPgD) scene.

Reviewer: Remove 1x `scene` because it's now duplicated in the sentence.
